api4/team_test.go

```diff
@@ -2479,16 +2226,10 @@ func TestInviteGuestsToTeam(t *testing.T) {
 			t.Log("No email was received, maybe due load on the server. Disabling this verification")
 		}
 		if err == nil && len(resultsMailbox) > 0 {
-			if !strings.ContainsAny(resultsMailbox[len(resultsMailbox)-1].To[0], email) {
-				t.Fatal("Wrong To recipient")
-			} else {
-				if resultsEmail, err := mailservice.GetMessageFromMailbox(email, resultsMailbox[len(resultsMailbox)-1].ID); err == nil {
-					if resultsEmail.Subject != expectedSubject {
-						t.Log(resultsEmail.Subject)
-						t.Log(expectedSubject)
-						t.Fatal("Wrong Subject")
-					}
-				}
+			require.True(t, strings.ContainsAny(resultsMailbox[len(resultsMailbox)-1].To[0], email), "Wrong To recipient")
+			resultsEmail, err := mailservice.GetMessageFromMailbox(email, resultsMailbox[len(resultsMailbox)-1].ID)
+			if err == nil {
```

Comment:
Probably here you can just use a `require.Nil(t, err)`

Reply:
The code here checks if err is nil then resultsEmail.Subject should be equal to expectedSubject else there is no failure and we move forward. Using require.Nil will fail is err is not nil.